src/com/amazon/ion/benchmark/schema/constraints/ByteLength.java

```diff
@@ -0,0 +1,23 @@
+package com.amazon.ion.benchmark.schema.constraints;
+
+import com.amazon.ion.IonValue;
+
+public class ByteLength extends QuantifiableConstraints{
+
+    /**
+     * Initializing the ByteLength object.
+     * @param value represents constraint field 'byte_length'.
+     */
+    public ByteLength(IonValue value) {
```

Comment:
> Since there's a public factory method, the constructor should be private. This goes for all of these constraint classes.

Thanks for catching this, will update them in the next commit.